torch/csrc/jit/serialization/mobile_bytecode_generated.h

```diff
@@ -4,9 +4,14 @@
 #ifndef FLATBUFFERS_GENERATED_MOBILEBYTECODE_TORCH_JIT_MOBILE_SERIALIZATION_H_
 #define FLATBUFFERS_GENERATED_MOBILEBYTECODE_TORCH_JIT_MOBILE_SERIALIZATION_H_
 
-C10_DIAGNOSTIC_PUSH_AND_IGNORED_IF_DEFINED("-Wnewline-eof")
 #include "flatbuffers/flatbuffers.h"
-C10_DIAGNOSTIC_POP()
+
+// Ensure the included flatbuffers.h is the same version as when this file was
+// generated, otherwise it may not be compatible.
+static_assert(FLATBUFFERS_VERSION_MAJOR == 23 &&
+              FLATBUFFERS_VERSION_MINOR == 3 &&
+              FLATBUFFERS_VERSION_REVISION == 3,
+             "Non-compatible flatbuffers version included");
```

Comment:
Yes, this will clearly break internal builds.
@osalpekar , @atalman mind importing it and landing via co-dev (with similar flatbuffers update)

Reply:
If I remember correctly, this might be OK. Because this file does not participate in internal builds. Internal generated header file is generated on the fly.